realm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,3 +4,4 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
+set JAVA_OPTS="-Xmx512m"
```

Comment:
I must confess I am not entirely sure about the difference (if any) between the two files, but if it doesn't make difference, then yes, it should probably be moved to the `gradle.properties` file instead